LIMBO server crash on startup due to required JSON format for motd

### Describe the problem

```
07:49:09 Info] Loading Limbo Version 0.6.9-ALPHA on Minecraft 1.18.1
Exception in thread "main" [07:49:10 Error]com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 4 path $
[07:49:10 Error]        at com.google.gson.Gson.assertFullConsumption(Gson.java:903)
[07:49:10 Error]        at com.google.gson.Gson.fromJson(Gson.java:893)
[07:49:10 Error]        at com.google.gson.Gson.fromJson(Gson.java:841)
[07:49:10 Error]        at com.google.gson.Gson.fromJson(Gson.java:813)
[07:49:10 Error]        at net.kyori.adventure.text.serializer.gson.GsonComponentSerializerImpl.deserialize(GsonComponentSerializerImpl.java:84)
[07:49:10 Error]        at net.kyori.adventure.text.serializer.gson.GsonComponentSerializerImpl.deserialize(GsonComponentSerializerImpl.java:37)
[07:49:10 Error]        at com.loohp.limbo.file.ServerProperties.<init>(ServerProperties.java:110)
[07:49:10 Error]        at com.loohp.limbo.Limbo.<init>(Limbo.java:170)
[07:49:10 Error]        at com.loohp.limbo.Limbo.main(Limbo.java:101)
[07:49:10 Error]Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 4 path $
[07:49:10 Error]        at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1568)
[07:49:10 Error]        at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1409)
[07:49:10 Error]        at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:542)
[07:49:10 Error]        at com.google.gson.stream.JsonReader.peek(JsonReader.java:425)
[07:49:10 Error]        at com.google.gson.Gson.assertFullConsumption(Gson.java:899)
[07:49:10 Error]        ... 8 more
2021-12-25T07:49:10.050Z        WARN    mc-server-runner        sub-process failed      {"exitCode": 1}
2021-12-25T07:49:10.050Z        INFO    mc-server-runner        Done
```

### Container definition

```
Paste run command or compose file here
```


### Container logs

```
Paste logs here
```

--- a/README.md
+++ b/README.md
@@ -587,6 +587,8 @@ Configuration options with defaults:
 - `LIMBO_SCHEMA_FILENAME`=default.schem
 - `LEVEL`="Default;${LIMBO_SCHEMA_FILENAME}"
 
+> NOTE: instead of using format codes in the MOTD, Limbo requires [JSON chat content](https://minecraft.fandom.com/wiki/Raw_JSON_text_format#Java_Edition). If a plain string is provided, which is the default, then it gets converted into the required JSON structure. 
+
 ### Running a Crucible server
 
 A [Crucible](https://github.com/CrucibleMC/Crucible) server can be run by setting `TYPE` to `CRUCIBLE`.

--- a/scripts/start-setupServerProperties
+++ b/scripts/start-setupServerProperties
@@ -48,7 +48,7 @@ function customizeServerProps {
 
   # If not provided, generate a reasonable default message-of-the-day,
   # which shows up in the server listing in the client
-  if [ -z "$MOTD" ]; then
+  if ! [ -v "$MOTD" ]; then
     # snapshot is the odd case where we have to look at version to identify that label
     if [[ ${ORIGINAL_TYPE} == "VANILLA" && ${VERSION} == "SNAPSHOT" ]]; then
       label=SNAPSHOT
@@ -60,6 +60,13 @@ function customizeServerProps {
     label=${label,,}
     label=${label^}
     MOTD="A ${label} Minecraft Server powered by Docker"
+  fi
+
+  # normalize MOTD
+  if [[ ${TYPE^^} = LIMBO ]]; then
+    if [[ $MOTD ]] && ! [[ $MOTD =~ ^{ ]]; then
+      MOTD="{\"text\":\"${MOTD}\"}"
+    fi
   fi
 
   setServerProp "server-name" SERVER_NAME

